app/frontend/src/wells/views/WellSearch.vue

```diff
@@ -289,38 +338,36 @@ export default {
             }
           })
         }
-      }).catch((e) => {
-        console.error(e)
       })
+    },
+    handleRouteChange () {
+      const query = this.$route.query
+      const emptyQuery = Object.keys(query).length === 0
+      const storeStateAsQS = this.buildQueryParams()
```

Comment:
It seems like the state is being handled through this method, so it would be nice if it's possible to have a unit test for it

Reply:
I have added a test that loads the component and triggers a `handleMapSearch` method on that component. That will force an update of the URL's query params calling `buildQueryParams`